tests/Unit/Artwork/Modules/InventoryManagement/Http/Requests/Category/UpdateCraftInventoryCategoryNameRequestTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Tests\Unit\Artwork\Modules\InventoryManagement\Http\Requests\Category;
+
+use Artwork\Modules\InventoryManagement\Http\Requests\Category\UpdateCraftInventoryCategoryNameRequest;
+use PHPUnit\Framework\TestCase;
+
+class UpdateCraftInventoryCategoryNameRequestTest extends TestCase
+{
+    public function testRules(): void
+    {
+        self::assertSame(
```

Comment:
In Tests lieber im `$this` context bleiben

Reply:
Gibt es da einen Grund für, wenn ja, welchen? Mein Blickpunkt ist: Am Ende wird eine statische Methode aufgerufen, ob mit self:: oder $this->

Reply:
Laut https://docs.phpunit.de/en/11.2/assertions.html gibt es keine "richtige" Art und weise wie man es schreibt. Mein Argument dafür ist einfach nur Konsistenz. Bei DataProvidern war es z.B. lange Zeit egal ob statisch oder nicht. Jetzt *müssen* sie `public static` sein